purview/models/distgit.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistgitCommit(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')  # commit_id
+    owner = RelationshipTo('.user.User', 'OWNED_BY')  # author
+    committer = RelationshipTo('.user.User', 'OWNED_BY')
```

Comment:
@ralphbean do you think the `committer` is unnecessary data? I think we only care about the author of the commit.

Reply:
If you keep the committer, you need to also keep the author *as* the author in addition to the author *as* the owner.

Let's keep all three.